RELEASE/scripts/sl_ascend.ash

```diff
@@ -12560,6 +12579,22 @@ boolean L9_oilPeak()
 	buffMaintain($effect[Fishy Whiskers], 0, 1, 1);
 	handleFamiliar("initSuggest");
 
+	// Force MCD usage
+	sl_change_mcd(11);
+
+	// Set baseline to force maximizer usage
+	set_property("sl_maximize_baseline", "1000ml 100max");
```

Comment:
Again, don't fiddle with this property automatically.

Reply:
I added this one (and the one previously) because maximize didn't seem to be setting anything at all until I added a value.

Reply:
An `addToMaximize` call didn't work? Did you have `sl_maximize_baseline` set to disabled or something? Either way you can't do this, since it would override a user's personal settings, if any.